### PR TITLE
docs: Student report(electronics)

### PR DESCRIPTION
**Pull request by pranshi112**
# Checklist

- [ ] fixes #xxxx
- [x] Passes `Semantic Pull Request` and `TypoCheck`
- [ ] New docs are listed in `README.md`
- [ ] New docs are listed in `index.md`
- [ ] Updated table of content
- [x] Updated nav links in mkdocs.yml

---

## PR description

Added Students' report(electronics).

### CODE REVIEW

File: docs/electronics/student_Reports/nidhi_yusra_2019_20.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- Knowledge about sensors and their working was understood.
+- Successfully performed simulations of several sensors in Proteus with Arduino (as Pi was unavailable there).
+- Successfully completed learning about PCB designing.
+- Successfully learnt about all the basics of python needed in raspberry pi coding.
+
+## FUTURE WORK
+
+1. More time would be invested to gain more hands-on experience with the Raspberry Pi environment.
+2. To learn interfacing of sensors using pi in proteus.
+3. To learn about the future developments that can be made in the rov to tackle different problems.
```
Comment (pnijhara) [inline docs/electronics/student_Reports/nidhi_yusra_2019_20.md]: Make it ROV

File: docs/electronics/student_Reports/nidhi_yusra_2019_20.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+4. Learning how to control speed and direction of dc motor.
+5. Designing battery management system.
+6. Learning more about the tether and power requirements for the rov.
```
Comment (pnijhara) [inline docs/electronics/student_Reports/nidhi_yusra_2019_20.md]: Make it ROV

File: mkdocs.yml [hunk 3]
```diff
@@ -18,7 +18,9 @@ nav:
     - Index: electronics/README.md
     - BMS Engineer: electronics/bms_Engineer.md
     - PCB Designer: electronics/pcb_Designer.md
-    - Student Reports: electronics/student_Reports/students.md
+    - Student Reports:
```
Comment (pnijhara) [inline mkdocs.yml]: Revert this for now.

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Review by pnijhara — APPROVED** [review]

**Comment by pnijhara** [comment]
Images should not contain any other information such as OS, personal name etc.
In some of the images, I can see uncropped SS is used which shows info about the OS(windows) and the watermark shows info about the mobile brand which we aren't promoting.

@syedjawadakhtar please write these points in the README also. Also, make SS mandatory as the camera photos downgrade the quality of document.

**Comment by syedjawadakhtar** [comment]
I think writing it every README, we can add this as a general rule in CONTRIBUTING.md
Rule for adding photos/diagrams etc.
Also I didn't get your last sentence... SS is better than photos?

**Comment by pnijhara** [comment]
> Also I didn't get your last sentence... SS is better than photos?

ScreenShots should be considered over camera photos.

**Pull request merged**